templates/main-page.html

```diff
@@ -1,24 +1,23 @@
-{% extends "auth.html" %}
-
-{% block body %}
+{% extends "auth.html" %} {% block body %}
 <div class="px-8 py-12 relative text-xl">
-    <a class="github-fork-ribbon left-top z-10" href="https://github.com/felienne/hedy" data-ribbon="Fork me on GitHub" title="Fork me on GitHub">Fork me on GitHub</a>
+    <a
+        class="github-fork-ribbon left-top z-10" href="https://github.com/felienne/hedy" data-ribbon="Fork me on GitHub" title="Fork me on GitHub">Fork me on GitHub</a
+    >
```

Comment:
These lines still have a single-standing `>` symbol, can this be on one line as well?

Reply:
sorry, mist that